data/styles/clear/include/Icons.mapcss

```diff
@@ -268,6 +268,8 @@ node[shop=supermarket],
 area[shop=supermarket],
 node[shop=car_repair][service=tyres],
 area[shop=car_repair][service=tyres],
+node[shop=caravan],
+area[shop=caravan],
 node[highway=services],
```

Comment:
Btw, is it important to list the tag in the z-index?
Because some tags are missing e.g. `shop=car`

Reply:
Well, current styles is something _shaky_ now. The best strategy is to find similar type and make the same. You did well, like shop=car.